packages/rrweb/test/utils.ts

```diff
@@ -356,11 +359,11 @@ export function stripBase64(events: eventWithTime[]) {
 
   return events.map((evt) => {
     if (
-      evt.type === EventType.IncrementalSnapshot &&
-      evt.data.source === IncrementalSource.CanvasMutation
+      evt.type === EventType.FullSnapshot ||
+      evt.type === EventType.IncrementalSnapshot
```

Comment:
@p-mazhnik why is it necessary to transform the event for all incremental and full snapshots?

Reply:
This utility is to strip base64 strings from snapshot in tests; Previous code didn't work for all instances where we have base64 strings in events, because of the lack of tests. We can have `rr_dataURL` base64 strings in full snapshots, because it is how canvas element is presented in full snapshot